sauce/features/budget/check-credit-balances/settings.js

```diff
@@ -1,29 +1,14 @@
 module.exports = {
   name: 'CheckCreditBalances',
   type: 'select',
-  default: false,
+  default: "disabled",
```

Comment:
This one is on me. The value for `default` should be true or false. In almost all cases it should be false because we don't have many features that are "defaulted to on".  Please forgive my brain fart. Doh!